spec/features/new_event_request_spec.rb

```diff
@@ -32,6 +34,13 @@
     page.should have_unchecked_field("coc")
   end
 
+  it "should have appropriate locations available" do
+    available_location = Location.available.first.name_with_chapter
+
+    page.should have_select('event_location_id', :with_options => [available_location])
+    page.should_not have_select('event_location_id', :with_options => [@archived.name_with_chapter])
```

Comment:
This assertion might be better written as a single `page.should have_select('event_location_id', :options => [])` that exhaustively lists all options to ensure that the `should_not` doesn't someday become a false positive.